app/controllers/servicedown_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ServicedownController < ActionController::Base
+  layout 'servicedown'
+
+  rescue_from StandardError do |exception|
+    puts 'StandardError caught'.yellow
+    render text: 'catchall'
+  end
+
+  def show
+    puts 'show action'.yellow
+    respond_to do |format|
+      format.html do
+        render :show
+      end
+      format.json do
+        render  json:
+                [{ error: 'Service temporarily unavailable' }],
+                status: :service_unavailable
+      end
+      format.js do
+        render  json:
+                [{ error: 'Service temporarily unavailable' }],
+                status: :service_unavailable
+      end
+      format.all do
+        render  plain: 'error: Service temporarily unavailable',
+                status: :service_unavailable
+      end
```

Comment:
(Rubocop - method length)
```suggestion
      format.html { render :show }
      format.json { render json: [{ error: 'Service temporarily unavailable' }], status: :service_unavailable }
      format.js { render json: [{ error: 'Service temporarily unavailable' }], status: :service_unavailable }
      format.all { render plain: 'error: Service temporarily unavailable', status: :service_unavailable }
```